apps/artwork/components/artists/test/fixture.coffee

```diff
@@ -0,0 +1,72 @@
+{ fabricate } = require 'antigravity'
+
```

Comment:
Also minor, I would rename this file. The difference between `artist_fixture` and `fixture` is not obvious on first glance, and any way you can save someone having to figure out the distinction in the future is 👍